collector/lib/ProcfsScraper.cpp

```diff
@@ -383,7 +384,12 @@ void ResolveSocketInodes(const SocketsByContainer& sockets_by_container, const C
         } else if (listen_endpoints) {
           if (const auto* ep = Lookup(ns_network_data->listen_endpoints, socket_inode)) {
             if (!IsRelevantEndpoint(ep->endpoint)) continue;
-            listen_endpoints->emplace_back(container_id, ep->endpoint, ep->l4proto, ep->process);
+
+            std::shared_ptr<Process> process;
+            if (process_store)
+              process = process_store->Fetch(ep->pid);
```

Comment:
[nit] Put braces around this or move it to the same line (braces preferred).
```suggestion
            if (process_store) {
              process = process_store->Fetch(ep->pid);
            }
```